src/cli/init.ts

```diff
@@ -81,10 +82,49 @@ export function builder(yargs: Argv) {
 }
 export async function handler(argv: Arguments) {
   let baseDir = argv.dir as string;
+  let packageInstalled: boolean = false;
   let ts = argv.typescript;
   let updateTsConfig = argv.tsconfig;
   let changes: [Logger, string][] = [];
 
+  try {
+    packageInstalled = await checkPackageJsonForNextVideo('./package.json');
```

Comment:
It could still not be installed yet even if it's in the package.json, right? Do we need to check for the node_module too/instead?

Reply:
In that case it would mean it was just installed locally, but this is explicitly all about installing it in their package.json, so this check should be sufficient there.